src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -19,23 +19,19 @@
 import seedu.address.commons.util.CollectionUtil;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
-import seedu.address.model.person.Remark;
+import seedu.address.model.client.*;
```

Comment:
This violates the coding standard, avoid using Java wildcard imports. Need to explicitly state the imports.